### PR TITLE
Clean up repeated capabilities tip

### LINKED ISSUE #854: Documentation shows outdated indexing of session in several places
In #718 things changed. We didn't update everything:

All drivers have this mistake:
```
docs/niscope/attributes.rst:            session['0,1'].accessory_gain = var
```
We should think carefully what we want to do.

niscope has:
```
docs/niscope/functions.rst:            wfm_info = session['0,1'].fetch_into(num_samples, waveform, timeout=5.0)
```

I think that's it.

### PR DESCRIPTION
**Pull request by texasaggie97**
- [X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).
- [x] ~~I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.~~
- [X] ~~I've added tests applicable for this pull request~~
### What does this Pull Request accomplish?
* Add repeated capability container information to nifgen (only driver that currently has non 'channels' containers)
* Add default repeated capability container information of 'channels' where appropriate
* Use repeated capability container information in tips
### List issues fixed by this Pull Request below, if any.
* Fix #854

### What testing has been done?
* Visual inspection of generated html and session files

### CODE REVIEW

File: docs/niswitch/class.rst [hunk 1]
```diff
@@ -1183,15 +1183,15 @@ max_carry_dc_current
...
+            niswitch.Session object, then the set/get will use all repeated capabilities in the session.
             You can specify a subset of repeated capabilities using the Python index notation on an
-            max_carry_dc_current.Session instance, and calling set/get value on the result.:
+            niswitch.Session repeated capabilities container, and calling set/get value on the result.:
 
             .. code:: python
 
-                session['0,1'].max_carry_dc_current = var
-                var = session['0,1'].max_carry_dc_current
+                session.channels[0,1].max_carry_dc_current = var
```
Comment (marcoskirsch) [inline docs/niswitch/class.rst]: On `niswitch`, it's pretty rare for channels to be 0,1... usually they are c0, r1, com2.
Most of these would error if called with two channels.

Comment (marcoskirsch) [reply docs/niswitch/class.rst]: Furthermore, a lot of these are read-only properties.
I think we should merge this PR but open an issue about this additional problem.

Comment (texasaggie97) [reply docs/niswitch/class.rst]: I do not know how to fix that, unless we add even more metadata somewhere.

Comment (texasaggie97) [reply docs/niswitch/class.rst]: True. I was thinking of the `0,1` part. I can fix the r/w/rw pretty easy.

Comment (texasaggie97) [reply docs/niswitch/class.rst]: Add r/w/rw specific tips

File: build/helper/documentation_helper.py [hunk 2]
```diff
@@ -121,6 +127,22 @@ def get_rst_admonition_snippet(node, admonition, d, config, indent=0):
...
 
+def add_attribute_rep_cap_tip_rst(attr, config):
+    '''Add the appropriate (r/w/rw/none) rst formatted tip for an attribute'''
+    if 'repeated_capability_type' in attr:
+        if attr['access'] == 'read only':
+            tip = rep_cap_attr_desc_rst_r
+        elif attr['access'] == 'write only':
+            tip = rep_cap_attr_desc_rst_w
+        else:
+            tip = rep_cap_attr_desc_rst_rw
```
Comment (marcoskirsch) [inline build/helper/documentation_helper.py]: assert it's read/write

Comment (texasaggie97) [reply build/helper/documentation_helper.py]: Added

**Pull request merged**